config.json

```diff
@@ -2279,6 +2279,32 @@
           "string-methods"
         ],
         "difficulty": 7
+      },
+      {
```

Comment:
This exercise is in the incorrect section of the `config.json`, and is failing CI.  It needs to be moved to the `concept exercise` section.

Reply:
No wonder it wasn't matching up with the others. Lol